driver/normalizer/normalizer.go

```diff
@@ -379,14 +426,13 @@ var Normalizers = []Mapping{
 		CommentNode(false, "text", nil),
 	)),
 
-	// FIXME: doesn't work
-	//MapSemantic("MultiLineCommentTrivia", uast.Comment{}, MapObj(
-		//Obj{
-			//uast.KeyToken: CommentText([2]string{"/*", "*/"}, uast.KeyToken),
-			//"IsDirective": Bool(false),
-		//},
-		//CommentNode(true, uast.KeyToken, nil),
-	//)),
+	MapSemantic("MultiLineCommentTrivia", uast.Comment{}, MapObj(
```

Comment:
How did you make this work, is because of the `Span` change above?

Reply:
Yes, it may be the `Span`, since some comment tokens where not containing tokens like `/*`.